use_cases/common/single_type/metrics/two_series_arithmetic.yaml

```diff
@@ -0,0 +1,71 @@
+title: Two Series Arithmetic
+
+description: >
+  A common operation operation involves combining two or more time series into a single
+  aggregate. Used most often in systems that require a single explicit metric name 
+  during query time with optional filters on tags. 
+
+inputs:
+  - metric series
+  -
+uses:
+  - type: Alerting
+    description: >
+      Used for simple, single threshold alerting on a combination of series. 
+
+  - type: Visualization
+    description: >
+      Used to reduce the amount of data displayed in a visualization and combine
+      complementary data.
+
+examples:
+  - language: PromQL
+    query: >
+      sum(http_requests_errors{handler="/api/comments"}) + 
+      sum(http_requests_success{handler="/api/comments"})
+    description: >
+      Selects two time series, `http_requests_errors` and `http_requests_success`,
+      filtering both where the tag `handler` has the value `/api/comments`, then
+      adding the results
+    references:
+      - https://prometheus.io/docs/prometheus/latest/querying/examples/#simple-time-series-selection
+      - https://prometheus.io/docs/prometheus/latest/querying/examples/#using-functions-operators-etc
+
+  - language: ASL
+    query: >
+      name,http_requests_errors,:eq,handler,/api/comments,:eq,:and,
+      name,http_requests_success,:eq,handler,/api/comments,:eq,:and,
+      :add
+    description: >
+      Selects two time series, `http_requests_errors` and `http_requests_success`,
+      filtering both where the tag `handler` has the value `/api/comments`, then
+      adding the results
+    references:
+      - https://netflix.github.io/atlas-docs/asl/ref/and/
+      - https://netflix.github.io/atlas-docs/asl/ref/eq/
+      - https://netflix.github.io/atlas-docs/asl/ref/add/
+
+  - language: ANSI SQL
+    query: >
+      TODO
+    description: >
+      Selects two time series, `http_requests_errors` and `http_requests_success`,
+      filtering both where the tag `handler` has the value `/api/comments`, then
+      adding the results
```

Comment:
Does the description need to be repeated for each example?

Reply:
good point, moved it to a top level field.